lib/rspec_logfmt_formatter/formatter.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'rspec/core'
+require 'rspec/core/formatters/base_formatter'
+
+module RspecLogfmtFormatter
```

Comment:
tiny bikeshed: `RspecLogfmtFormatter` vs `RSpecLogfmtFormatter`? Not sure if there's a convention in namespaces for RSpec related projects.

Reply:
I had a look at the [rspec_junit_formatter](https://github.com/sj26/rspec_junit_formatter) and they use `RspecJunitFormatter` to call it but `RSpecJunitFormatter` as the class name. So I might leave it for now and see if I can find some other examples.